### PR TITLE
HRINT-2697 - FTS Query - Using Wildcards

### PR DESCRIPTION
**Pull request by Drorlev**
FTS Query - Using Wildcards

### CODE REVIEW

**Review by ml054 — APPROVED** [review]

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+                    "Type": "demo",
+                    "Url": "/java/queries/query-example#step-5"
+                }
+            ]
+        },
+        {
+            "Title": "Execute the query",
+            "Slug": "step-5",
+            "DescriptionLines": [
+                "* Execute the query by calling `ToList()`.",
```
Comment (Danielle9897) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json:159]: ToList() => toList()

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+                {
+                    "Title": "How to search",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/java/client-api/session/querying/how-to-use-search"
+                },
+                {
+                    "Title": "Querying searching",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/java/indexes/querying/searching"
+                }
```
Comment (Danielle9897) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json:94]: Are the above 2 links working ? 
or do you want to add 'TODO' here for them, as was done in walkthrough step number 3 ?

**Review by Danielle9897 — CHANGES_REQUESTED** [review]

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json (hunk 1 above)
Comment (Drorlev) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json:159]: Done

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+        "* both prefix & postfix</p>",
+        "",
+        "Upon query, an auto-index is created on the searched document-fields."
+    ],
+
+    "Assets": [
+        {
+            "Title": "FTS Query - Basics Demo",
+            "Type": "demo",
+            "Url": "/java/text-search/fts-query-search-basics"
```
Comment (Danielle9897) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json:21]: As discussed, align the `':'`

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json [hunk 4]
```diff
@@ -100,9 +100,9 @@
         },
         "demos": [
             "fTSQuerySearchBasics",
+            "fTSQuerySearchWildcards",
             "fTSQuerySearchBoosting",
-            "fTSWithStaticIndexSingleField",
-            "fTSWithStaticIndexMultipleFields"
+            "fTSWithStaticIndexSingleField",            "fTSWithStaticIndexMultipleFields"
```
Comment (Danielle9897) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json:105]: why on the same line ?

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json (hunk 2 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json:95]: @Drorlev 
For the future:
Pls write `'done'` for any comment when you apply the fix.

**Review by Danielle9897 — CHANGES_REQUESTED** [review]

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json (hunk 3 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json:21]: done

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json (hunk 4 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json]: done

**Review by Danielle9897 — APPROVED** [review]

**Comment by Danielle9897** [comment]
@ml054 this can be merged

**Pull request merged**